### PR TITLE
+ simple status reporting

### PR DESCRIPTION
**Pull request by raymondng**
very simple impl -- mostly boilerplate code in the pull request

up next: admin page wire-in

with this, we can move away from thread-id based reporting which places unnecessary constraints on how scraper is implemented internally

@yingjieMiao @martinraison @ymatsuda @leogrim @eishay

### CODE REVIEW

File: kifi-backend/modules/scraper/app/com/keepit/scraper/ScrapeProcessor.scala [hunk 1]
```diff
@@ -2,11 +2,27 @@ package com.keepit.scraper
...
+  import play.api.libs.functional.syntax._
+  import play.api.libs.json._
+
+  implicit val format = (
+    (__ \ 'worker).format[String] and
+    (__ \ 'submitTS).format[DateTime] and
+    (__ \ 'uri).format[NormalizedURI] and
+    (__ \ 'scrape).format[ScrapeInfo]
+  )(ScrapeJobStatus.apply _, unlift(ScrapeJobStatus.unapply))
 }
```
Comment (yingjieMiao) [inline kifi-backend/modules/scraper/app/com/keepit/scraper/ScrapeProcessor.scala]: maybe we can try the @json macro? @martinraison

File: kifi-backend/modules/common/app/com/keepit/model/ScrapeJobStatus.scala [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  import play.api.libs.functional.syntax._
+  import play.api.libs.json._
+
+  implicit val format = (
+    (__ \ 'worker).format[String] and
+    (__ \ 'submitTS).format[DateTime] and
+    (__ \ 'uri).format[NormalizedURI] and
+    (__ \ 'scrape).format[ScrapeInfo]
+  )(ScrapeJobStatus.apply _, unlift(ScrapeJobStatus.unapply))
+}
```
Comment (yingjieMiao) [inline kifi-backend/modules/common/app/com/keepit/model/ScrapeJobStatus.scala:17]: duplicated class? common and scraper project both have this?

**Comment by yingjieMiao** [comment]
a few minor comments. o.w. lgtm

File: kifi-backend/modules/common/app/com/keepit/model/ScrapeJobStatus.scala (hunk 2 above)
Comment (raymondng) [inline kifi-backend/modules/common/app/com/keepit/model/ScrapeJobStatus.scala:17]: oops! thanks and fixed

**Pull request merged**